### PR TITLE
Not necessary globals removed to avoid multiple definition linker errors

### PR DESCRIPTION
**Pull request by jgoreckiavs**

### CODE REVIEW

File: source/include/cellular_setup.h [hunk 1]
```diff
@@ -13,10 +13,10 @@
 
 /* Secure socket needs application to provide the cellular handle and pdn context id. */
 /* User of secure sockets cellular should provide this variable. */
-CellularHandle_t CellularHandle;
+extern CellularHandle_t CellularHandle;
 
 /* User of secure sockets cellular should provide this variable. */
-uint8_t CellularSocketPdnContextId;
+extern uint8_t CellularSocketPdnContextId;
```
Comment (ferre111) [inline source/include/cellular_setup.h]: We can get rid of them completely.

**Review by ferre111 — COMMENTED** [review]
@jgoreckiavs Looks good, but there are new warnings. Most of them can be simply removed by using types from `inttypes`.

File: README.md [hunk 2]
```diff
@@ -1,133 +1,22 @@
...
+[FreeRTOS Cellular Interface v1.2.0](https://github.com/FreeRTOS/FreeRTOS-Cellular-Interface/tree/v1.2.0)<br>
+[Free RTOS Cellular Demo](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Demo)<br>
+    `cellular_platform.c`<br>
+    `cellular_platform.h`<br>
+    `cellular_setup.c`<br>
+    `logging_levels.h`<br>
+    `logging_stack.h`<br>
+[FreeRTOS](https://github.com/FreeRTOS/FreeRTOS/tree/main/FreeRTOS-Plus/Source/Application-Protocols/network_transport/sockets_wrapper/cellular)<br>
+    `sockets_wrapper.c`<br>
+    `sockets_wrapper.h`<br>
```
Comment (ferre111) [inline README.md]: Please consider this version:

```
This project, in addition to being a fork of the `FreeRTOS Cellular Interface`, also relies on slightly modified files from the following projects:

[Free RTOS Cellular Demo](https://github.com/FreeRTOS/Lab-Project-FreeRTOS-Cellular-Demo)<br>
* `cellular_config.h`<br>
* `cellular_platform.c`<br>
* `cellular_platform.h`<br>
* `cellular_setup.c`<br>
* `logging_levels.h`<br>
* `logging_stack.h`<br>

[FreeRTOS](https://github.com/FreeRTOS/FreeRTOS/tree/202112.00/FreeRTOS-Plus/Source/Application-Protocols/network_transport/sockets_wrapper/cellular)<br>
* `sockets_wrapper.c`<br>
* `sockets_wrapper.h`<br>
```

**Review by barwoj01 — APPROVED** [review]
LGTM

**Pull request merged**